### ISSUE TITLE
Audio/podcast?

### ISSUE THREAD

**Issue by carols10cents** [rust-lang/blog.rust-lang.org#1447]
Say I wanted to record myself reading a blog post that I also posted in text form on the blog so that people could listen to it in podcast form.

I am _not_ advocating that every blog post do this, just that I have some desire to do so for [my Foundation Board series](https://github.com/rust-lang/blog.rust-lang.org/pull/1445), as an experiment to see if adding different mediums increases the amount that the community feels informed of Foundation activities.

I have a bunch of questions for discussion, and there are probably more concerns I haven't thought of:

1. Does RSS have a mechanism for selectively identifying an item as a podcast episode, such that someone could put the Inside Rust RSS feed in their podcast app and see only those posts that have audio "attached"?
2. Would storing audio files in this repo make it annoying to clone and work with or otherwise hit GitHub service restrictions?
3. Alternatively, if I do this entirely out-of-band on my own infra/domain/etc, would anyone have a problem with that? Would it be weird to have the "official" blog post link to the "unofficial" podcast form?

Another note about my broader plans: for the Board meeting minutes highlight posts, I'm definitely planning on composing the written form first and then potentially recording the audio. In the future, I have ideas on doing Project Director updates that might be better as audio first, such as interviews with Foundation grant recipients on the work they've done. In those cases, I would want to record the podcast first and then get it transcribed. But because my goal is to increase the mediums used to communicate, my plan is always to have _both_ written _and_ audio for people to choose from.

**Comment by bjorn3** [comment]
Based on https://github.com/Podcast-Standards-Project/PSP-1-Podcast-RSS-Specification it looks like `<itunes:block>yes</itunes:block>` can be used to hide an entry from itunes and other apps compatible with the podcast format used by itunes.

**Comment by CleanCut** [comment]
> Would storing audio files in this repo make it annoying to clone and work with...

Yes, it can be annoying to work with repositories with large audio files. Every person who ever clones the repo will have to download all audio files ever added to the repository.

> or otherwise hit GitHub service restrictions?

No, it probably would not hit GitHub service restricitons. I anticipate your audio files would be in some compressed format and be less than 50MB. If they were ~5MB each, you probably wouldn't hit 1GB for many years. See the [About large files on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github) page for specific guidelines and limits. 

My recommendation would be to store the audio files somewhere else, and point to them from here. Even a separate GitHub repo dedicated to storing the files would work decently. Storing them in some cloud service would probably be even better. (SoundCloud? -- I'm not an audio guy, I dunno what the options are)

**Comment by bjorn3** [comment]
Git LFS on Github has a somewhat limited maximum amount of download bandwidth that can be used per month without paying extra. On the free plan that is for example 10GB. No idea how much it is for the rust-lang org though.

**Comment by senekor** [comment]
There was a good blog post recently about the problems of Git LFS:
https://tylercipriani.com/blog/2025/08/15/git-lfs/
